apps/gitness/src/pages-v2/project/project-rules-list-container.tsx

```diff
@@ -46,7 +48,10 @@ export const ProjectRulesListContainer = () => {
     space_ref: `${space_ref}/+`,
     queryParams: {
       page: queryPage,
-      query: query ?? ''
+      query: query ?? '',
+      inherited: showParentRules,
+      // @ts-expect-error BE expects an array but the API only works with a string
+      type: ruleTypeFilter ? ruleTypeFilter : undefined
```

Comment:
Is this not resolved with the latest version of code-service client as well? That's the most up-to-date version we have. We should raise this with backend team as a follow up task.